packages/checkbox/src/Checkbox.js

```diff
@@ -1,6 +1,98 @@
-import { Checkbox } from "hig-react";
-import "./checkbox.scss";
+import React, { Component } from "react";
+import PropTypes from "prop-types";
 
-Checkbox.displayName = "Checkbox";
+import CheckboxPresenter from "./CheckboxPresenter/CheckboxPresenter";
 
-export default Checkbox;
+export default class Checkbox extends Component {
+  static propTypes = {
+    /**
+     * Checks the checkbox
+     */
+    checked: PropTypes.bool,
+    /**
+     * Initially checks the checkbox, but allows user action to change it
+     */
+    defaultChecked: PropTypes.bool,
+    /**
+     * Prevents user actions on the checkbox
+     */
+    disabled: PropTypes.bool,
+    /**
+     * Sets indeterminate state for checkbox
+     */
+    indeterminate: PropTypes.bool,
+    /**
+     * Text identifying the field
+     */
+    label: PropTypes.string,
+    /**
+     * The name of the checkbox as submitted with a form
+     */
+    name: PropTypes.string,
+    /**
+     * Called when user moves focus from the field
+     */
+    onBlur: PropTypes.func,
+    /**
+     * Called when user changes the value of the field
+     */
+    onChange: PropTypes.func,
+    /**
+     * Called when user puts focus on the field
+     */
+    onFocus: PropTypes.func,
+    /**
+     * Marks the field as required, text shown to explain requirement
+     */
+    required: PropTypes.string,
+    /**
+     * Value submitted with a form if checked
+     */
+    value: PropTypes.string
+  };
+
+  state = {
+    checked: false
+  };
+
+  static defaultProps = {
+    defaultChecked: true
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: props.defaultChecked || Checkbox.defaultProps.defaultChecked
```

Comment:
There's no need to access `defaultProps` directly, the defaults will be applied `this.props` by the time you access them.

Reply:
Updated in 8c8d0a4f